docs/conf.py

```diff
@@ -119,6 +119,7 @@
     # Your documentation GitHub repository URL
     "github_url": "https://github.com/juju/juju",
     # Docs branch in the repo; used in links for viewing the source files
+    "github_issues": "https://github.com/juju/juju/issues",
```

Comment:
The comment is now for the wrong line

You should move line 121 down a line, and write a quick comment explaining the `github_issues` key

Reply:
Updated.